<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>housedialog</class>
 <widget class="QDialog" name="housedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="nameEdit">
     <property name="text">
      <string>仓库名称</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="widthEdit">
       <property name="text">
        <string>纵向长度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="heightEdit">
       <property name="text">
        <string>横向宽度</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="xposEdit">
       <property name="text">
        <string>集散点纵向位置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yposEdit">
       <property name="text">
        <string>集散点横向位置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="desEdit">
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;仓库描述。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>housedialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
